crates/matrix-sdk-search/src/index.rs

```diff
@@ -184,6 +190,18 @@ impl RoomIndex {
 
         Ok(ret)
     }
+
+    fn contains(&self, event_id: &EventId) -> bool {
+        let search_result = self.search(format!("event_id:\"{event_id}\"").as_str(), 5);
+        match search_result {
+            // If there are > 1 entry (which there shouldn't be), still return true
```

Comment:
I don't understand this comment: why "still" return true? (also, missing dot at the end)